yt_dlp/cookies.py

```diff
@@ -164,30 +164,17 @@ def _extract_firefox_cookies(profile, logger):
 
 
 def _firefox_browser_dir():
-    if sys.platform in ('linux', 'linux2'):
-        return os.path.expanduser('~/.mozilla/firefox')
-    elif sys.platform == 'win32':
+    if sys.platform == 'win32':
```

Comment:
while we are editing this, cygwin should also be supported
```suggestion
    if sys.platform in ('win32', 'cygwin'):
```